cmd/frostfs-adm/internal/modules/morph/root.go

```diff
@@ -323,4 +335,8 @@ func init() {
 	depositNotaryCmd.Flags().String(walletAccountFlag, "", "Wallet account address")
 	depositNotaryCmd.Flags().String(refillGasAmountFlag, "", "Amount of GAS to deposit")
 	depositNotaryCmd.Flags().String(notaryDepositTillFlag, "", "Notary deposit duration in blocks")
+
+	RootCmd.AddCommand(netmapCandidatesCmd)
+	netmapCandidatesCmd.Flags().StringP(endpointFlag, "r", "", "N3 RPC node endpoint")
+	netmapCandidatesCmd.Flags().BoolVar(&short, "short", false, "Shortens output of node info")
```

Comment:
Why not short by default and `--verbose` for everything?

Reply:
Good idea, replaced with `--verbose`.